### ISSUE TITLE
Working?

### ISSUE THREAD

**Issue by robertjchristian** [rorymadden/angular-login#1]
Looks like an interesting and useful project.  Did you ever get it working?  Looks like it's no longer being maintained.  Thanks.

**Comment by rorymadden** [comment]
No, this was never working unfortunately. I stopped using MongoDB as a database (working with Neo4j) and I found this project https://github.com/angular-app/angular-app which seems to give most of the functionality, except registration. I'm travelling at the moment but when I get back in June I might give this another look. Apologies.

**Comment by robertjchristian** [comment]
Thanks for the update! rorymadden notifications@github.com wrote:No, this was never working unfortunately. I stopped using MongoDB as a database (working with Neo4j) and I found this project https://github.com/angular-app/angular-app which seems to give most of the functionality, except registration. I'm travelling at the moment but when I get back in June I might give this another look. Apologies.
